lib/static/components/section/title/browser.jsx

```diff
@@ -22,7 +22,7 @@ const BrowserTitle = (props) => {
             viewModes: ViewMode.ALL,
             expand: EXPAND_ALL
         });
-    };
+    }, [window.location.href, props.browserName, props.testName, props.retryIndex]);
```

Comment:
`window.location.href` is not reactive as far as i know. So placing it in dependencies will do nothing.